demos/sports-analytics-architecture/main.bicep

```diff
@@ -0,0 +1,252 @@
+@description('Resource Location.')
+param location string = resourceGroup().location
+
+@description('Your Azure AD user identity (this identity will be granted admin rights to the Azure SQL instance).')
+param principalName string
+
+@description('Object ID for your Azure AD user identity (see the README.md file in the Azure Quickstart guide for instructions on how to get your Azure AD user object ID).')
+param principalObjectId string
+
+@description('Name of the Azure Data Lake Storage Gen2 account')
+param adlsAccountName string
+
+@description('Name of the Azure Data Factory instance')
+param adfName string
+
+@description('Name of the Azure Databricks workspace')
+param azureDatabricksName string
+
+@description('Do you want to deploy a new Azure Event Hub for streaming use cases? (true or false)')
+param deployEh bool
+
+@description('Name of the Azure Event Hub')
+param eventHubName string
+
+@description('Do you want to deploy a new Azure SQL Database (true or false)?')
+param deploySqlDb bool
+
+@description('Do you want to enable No Public IP (NPIP) for your Azure Databricks workspace? (true or false)')
+param databricksNpip bool
+
+@description('Do you want to deploy a new Azure Key Vault instance? (true or false)')
+param deployAkv bool
+
+@description('Name of the Azure Key Vault')
+param akvName string
+
+@description('Name of Azure SQL logical server')
+param azureSqlServerName string
+
+@description('Name of the SQL Database')
+param azureSqlDatabaseName string
+
+@description('SQL administrator Username')
+param sqlAdministratorLogin string
+
+@description('SQL administrator Password')
+@secure()
+param sqlAdministratorLoginPassword string
+
+var akvRoleName = 'Key Vault Secrets User'
+
+var akvRoleIdMapping = {
+  'Key Vault Secrets User': '4633458b-17de-408a-b874-0445c86b69e6'
+}
+
+resource eventHubNamespace 'Microsoft.EventHub/namespaces@2021-11-01' = if(deployEh) {
```

Comment:
@bmoore-msft I'm a little confused by this statement. Do you want me to set the boolean params to true? Or something else?

Reply:
Yes, set the values to `true` so that these resource deployments are tested...

Reply:
Got it. When I set the deploySqlDb param to true and the template deploys an instance of Azure SQL DB, the template will also enable AAD authentication and set an AAD admin using the principalName param. Since I'm passing a dummy value for the test (like TestAADAdminName), will the test fail?

Reply:
Scratch that, I made the changes and everything went through without a hitch.